css/style.css

```diff
@@ -0,0 +1,267 @@
+    font-family: Superclarendon, sans-serif;
+    text-shadow: -1px 0 #000000, 0 1px #000000, 1px 0 #000000, 0 -1px #000000;
+}
+
+nav {
+    padding: 10px;
+    text-align: center;
+}
+
+.hero {
+    height: 500px;
+    background-image: url(../img/hero.jpg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+}
+
+.title {
+    text-align: center;
+    color: #EC7505;
+    font-size: 2rem;
+    font-family: Superclarendon, sans-serif;
+    text-shadow: -1px 0 #000000, 0 1px #000000, 1px 0 #000000, 0 -1px #000000;
+}
+
+.heading {
+    font-size: 1.7rem;
+    text-align: center;
+}
+
+.heading h1 {
+    margin: 0 auto;
+    font-size: 8rem;
+    color: #CC3F0C;
+    font-family: SignPainter, sans-serif;
+    text-shadow: -1px 0 #000000, 0 1px #000000, 1px 0 #000000, 0 -1px #000000;
+}
+
+.heading h2 {
+    margin: 0 auto;
+    background-color: #FAA300;
+    font-family: Superclarendon, sans-serif;
+    letter-spacing: 0.1rem;
+    padding: 18px;
+}
+
+.portfolioTitle {
+    height: 170px;
+    margin: 0 auto;
+    color: #CC3F0C;
+    text-align: center;
+    font-family: SignPainter, sans-serif;
+    font-size: 8rem;
+    text-shadow: -1px 0 #000000, 0 1px #000000, 1px 0 #000000, 0 -1px #000000;
+}
+
+.projectBox {
+    float: left;
+    width: 33.3%;
+    height: 250px;
+}
+
+.projectInfo {
+    text-align: center;
+    opacity: 0;
+    color: #D12411;
+    text-shadow: -1px 0 #000000, 0 1px #000000, 1px 0 #000000, 0 -1px #000000;
+    font-family: "Lato", sans-serif;
+    font-size: 1rem;
+    letter-spacing: 0.2em;
+}
+
+.projectInfo:hover {
+    opacity: 100;
+}
+
+.project1 {
+    background-image: url(../img/project1.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 390px;
+}
+
+.project2 {
+    background-image: url(../img/project2.png);
+    background-position: center;
+    background-size: 490px;
+}
+
+.project3 {
+    background-image: url(../img/comingSoon.png);
+    background-position: center;
+    background-size: cover;
+}
+
+.project4 {
+    background-image: url(../img/comingSoon.png);
+    background-position: center;
+    background-size: cover;
+}
+
+.project5 {
+    background-image: url(../img/comingSoon.png);
+    background-position: center;
+    background-size: cover;
+}
+
+.project6 {
+    background-image: url(../img/comingSoon.png);
+    background-position: center;
+    background-size: cover;
+}
+
+.contactInfo {
+    font-size: 4.3rem;
+    color: #CC3F0C;
+    text-align: center;
+    font-family: SignPainter, sans-serif;
+    text-shadow: -1px 0 #000000, 0 1px #000000, 1px 0 #000000, 0 -1px #000000;
+}
+
+.contactTitle {
+    margin: 0 auto;
+}
+
+.contactSymbols {
+    text-align: center;
+    height: 100px;
+    background-color: #577399;
+}
+
+.contactSymbols ul {
+    list-style: none;
+}
+
+.contactSymbols li {
+    display: inline-block;
+}
+
+.icon {
+    margin: 15px;
+    height: 70px;
+    width: 70px;
```

Comment:
height and width: don't remove yet but the size of the icon is being controlled at some other attributes. could be useful in later edits.

Reply:
Changing both does effect the size of the icons